client/components/reviewStars.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+
+const ReviewStars = ({rating}) => {
+  const convertToWidth = rating => {
+    return `${Math.round(rating / 5 * 100 / 10) * 10}%`
```

Comment:
Could you add some parentheses to show what operations happen in what order for this calculation?

Reply:
ok, put parentheses there. that math thing gives us a string like '90%' '80%' '70%'...  but not '75%'